src/App/Pages/Vault/CipherAddEditPageViewModel.cs

```diff
@@ -307,6 +307,7 @@ public bool HasCollections
         public bool PasswordPrompt => Cipher.Reprompt != CipherRepromptType.None;
         public string PasswordVisibilityAccessibilityText => ShowPassword ? AppResources.PasswordIsVisibleTapToHide : AppResources.PasswordIsNotVisibleTapToShow;
         public bool HasTotpValue => IsLogin && !string.IsNullOrEmpty(Cipher?.Login?.Totp);
+        public bool AllowTotpCopy => HasTotpValue && (Cipher?.ViewPassword ?? false);
```

Comment:
:art: No need for parenthesis and IMO equal to true is more readable.
```suggestion
        public bool AllowTotpCopy => HasTotpValue && Cipher?.ViewPassword == true;
```
Aside from this tiny fix, should we raise property changed when the `Cipher` property changes? I ask because if `Cipher` starts with `null` then if `AllowTotpCopy` gets evaluated will be `false` and when `Cipher` is loaded currently it won't trigger `AllowTotpCopy` property changed so there will be an inconsistency if `ViewPassword` is `true` afterwards. Thoughts?

Reply:
Upon closer inspection it looks like the null check isn't necessary so I just removed it.

Reply:
@mpbw2 I'd add it to prepare for MAUI or we're going to have issues when we merge this change to the maui branch because of how the binding engine works.

Reply:
@mpbw2 oh and isn't the `RaisePropertyChanged` necessary when `Cipher` changes?

Reply:
> @mpbw2 oh and isn't raising property changed necessary when `Cipher` changes?

It doesn't appear so - I'm seeing the expected results on test items containing both scenarios (tested on both iOS & Android).  In addition I don't believe changes to `Cipher` are possible beyond the initial load while on the Add/Edit screen, but correct me if I'm missing something.